adapters/flashbots/signature.go

```diff
@@ -0,0 +1,67 @@
+// Package flashbots provides methods for parsing the X-Flashbots-Signature header.
```

Comment:
Tests for this would be great 🤠

Reply:
Added in https://github.com/flashbots/rpc-endpoint/pull/151/commits/e4f8b3119d455f0ea0e8bc75d3859a883c496e76!